src/lib/Constants.js

```diff
@@ -95,8 +95,8 @@ export const APP_PRIORITIES = {
 	SUBSCRIBE_TO_TOPICS: 750,
 	IMAGE_UPLOAD: 700,
 	COUNT: 650,
-	SCORE: 640,
 	TIMES_VALIDATION3: 550,
+	SCORE: 540,
```

Comment:
while it doesn't have an impact on this, lets try to ensure that all modules run before the the final time validation module.